beacon-node-oapi.yaml

```diff
@@ -438,6 +438,10 @@ components:
       $ref: './types/electra/consolidation.yaml#/Electra/PendingConsolidation'
     Electra.PendingPartialWithdrawal:
       $ref: './types/electra/withdrawal.yaml#/Electra/PendingPartialWithdrawal'
+    Fulu.BlockContents:
```

Comment:
I think we also need a `Fulu.BlobSidecars` for the `getBlobSidecars` endpoint? Could be in a separate PR though.

Reply:
Yeah, since max blob number is not determined, prefer to do it later

Reply:
Blob count is not the only thing that changes - each blob sidecar has a KzgProof field that should be changed to plurals with cell proofs. I reckon we could set a high but realistic blob count for the current PeerDAS, to avoid having to update this endpoint on every BPO hard fork?

Reply:
cool, will update it tomorrow with other changes

Reply:
> I think we also need a Fulu.BlobSidecars for the getBlobSidecars endpoint? Could be in a separate PR though.

or we could deprecate the current one and at a new `getDataColumnSidecars`? I feel like this is something consumers should give feedback on before doing something, afaik, L2 clients use those apis

Reply:
I think we'd  probably want `getBlobsSidecarV2` for blob consumers (L2s). `DataColumnSidecars` is unlikely useful to end users - it's a CL-specific structure used for sample distribution and requires extra effort to convert into blobs, which L2 users are unlikely interested in. There could be use cases I'm missing though.

Reply:
> I think we'd probably want getBlobsSidecarV2 for blob consumers (L2s)

how would a CL be able to reconstruct the blob if it's not a supernode? and is there any reason why we need a v2 api, the current api is fork-aware so can change the type if required

Reply:
Let's kick off the discussion on getBlobsSidecar/V2 api, let's do it in a separate PR to speed up merging this one